services/galley/src/Galley/API/Internal.hs

```diff
@@ -396,9 +405,13 @@ type ITeamsAPIBase =
 
 type IFeatureStatusGet f = Named '("iget", f) (FeatureStatusBaseGet f)
 
-type IFeatureStatusPut errs f = Named '("iput", f) (FeatureStatusBasePutInternal errs f)
+type family FoldFederatedCalls (calls :: [*]) r where
+  FoldFederatedCalls '[] r = r
+  FoldFederatedCalls (call ': calls) r = call :> FoldFederatedCalls calls r
```

Comment:
And again, this is the same as the `FoldSegments` and `ApplyMods` type families, isn't it? Seeing it for the third time I'm convinced this ought to be written only once and then just used.